### DISCUSSION TITLE
Private Internet Access & 2FA?

### DISCUSSION THREAD

**Discussion by delipastrami** [qdm12/gluetun#1139] [Ideas]
Hello,

I was wondering if there is a method to configure gluetun with PIA vpn if the account has 2FA enabled? At the moment mine does not and so the container variable "OPENVPN_PASSWORD" works just fine with pass only. Is it possible to configure gluetun to accept a TOTP somehow?

Thank you

**Comment by qdm12** [comment]
Can you try turning on 2fa and then set OPENVPN_PASSWORD=yourpassword:yourotp and see if it works?

If it doesn't work please create an issue 😉

**Reply by delipastrami** [reply]
will do, thank you!
